internal/docstore/drivertest/drivertest.go

```diff
@@ -302,9 +302,7 @@ func testCodec(t *testing.T, ct CodecTester) {
 	type S struct {
 		N  *int
 		I  int
-		U  uint
```

Comment:
What is the implication of removing these from the test? I thought the idea was to have all types supported by `docstore` here, and do things in the drivers to make them support them (e.g., list of size 2 for complex).

If you're removing them because firestore doesn't have native support, then we're going to end up only testing the intersection of all types supported by all drivers; we need a better way.

Reply:
I guess one way to do it is to implement a driver codec that converts these types to structs. But I think the challenge might be that the native firestore client won't recognize them and we do want both the native and the docstore codec to work?

Reply:
As per our discussion, I will leave these out in this PR. In a followup PR, I'll (a) have a Docstore round-trip test with a full set of types, and (b) add an "I don't support these types" method to `CodecTester` so providers can opt out of certain types for the native-docstore tests.

Reply:
SGTM.